parsl/executors/high_throughput/executor.py

```diff
@@ -20,6 +20,10 @@
 from parsl.executors.errors import BadMessage, ScalingFailed
 from parsl.executors.high_throughput import zmq_pipes
 from parsl.executors.high_throughput.errors import CommandClientTimeoutError
+from parsl.executors.high_throughput.manager_selector import (
+    ManagerSelectorBase,
+    ManagerSelectorRandom,
```

Comment:
this naming could be more in line with the rest of the parsl abstractions:

The base names the think (`ExecutionProvider`, `Launcher`, `Channel`) - so that when you write code/docs you say "this argument is a `Launcher` and this other argument is a `Channel` or `provider: ExecutionProvider" - using `ManagerSelector` without `Base` would fit with that.

The other abstractions in Parsl put the adjective-like bit at the start of name for the concrete user facing instances so maybe this should be `RandomManagerSelector` to be consistent with that - for example:
```
parsl/providers/__init__.py:__all__ = ['LocalProvider',
parsl/providers/__init__.py:           'CobaltProvider',
parsl/providers/__init__.py:           'CondorProvider',
parsl/providers/__init__.py:           'GridEngineProvider',
parsl/providers/__init__.py:           'SlurmProvider',
parsl/providers/__init__.py:           'TorqueProvider',
parsl/providers/__init__.py:           'LSFProvider',
parsl/providers/__init__.py:           'AdHocProvider',
parsl/providers/__init__.py:           'PBSProProvider',
parsl/providers/__init__.py:           'AWSProvider',
parsl/providers/__init__.py:           'GoogleCloudProvider',
parsl/providers/__init__.py:           'KubernetesProvider',
parsl/providers/__init__.py:           'AzureProvider']
```

```
class SingleNodeLauncher(Launcher):
class GnuParallelLauncher(Launcher):
class MpiExecLauncher(Launcher):
class MpiRunLauncher(Launcher):
class SrunLauncher(Launcher):
class SrunMPILauncher(Launcher):
    Use this launcher instead of SrunLauncher if each block will execute multiple MPI applications
class AprunLauncher(Launcher):
class JsrunLauncher(Launcher):
```

Reply:
New commit changes the name of the ABC and its subclasses to the suggested naming scheme.